proto/content_analysis/sdk/analysis.proto

```diff
@@ -8,24 +8,24 @@ option optimize_for = LITE_RUNTIME;
 
 package content_analysis.sdk;
 
-// Handshake enum is sent with every message indicating the
-// status of the issuing browser or the agent.
+// A Handshake is sent when opening a new connection from browser to agent.
 message Handshake {
-  enum Status {
-    FAILURE = 0;
-    SUCCESS = 1;
-  }
-  optional Status status = 1; 
+  // Process ID of the browser initiating the session.
+  optional int32 process_id = 1;
+  // Indicates whether a content analysis request will be sent in this session,
+  // following the handshake. If set to True, then the browser is expecting to
+  // receive a verdict response from the agent in this session, following
+  // the request. If set to False, the browser is performing a startup check or
+  // an intermittent heartbeat, and no response is expected from the agent.
+  optional bool content_analysis_request = 2;
```

Comment:
Nit: you might want to add an adjective or a past participles to the end for clarification. See https://g3doc.corp.google.com/engage/g3doc/standards/protos.md?cl=head#field_names

e.g. content_analysis_request ->  content_analysis_request_sent

Reply:
Thanks for the link to the proto naming convenctions, havent' nseen them before!

Changed it to content_analysis_requested.